packages/block-editor/src/store/selectors.js

```diff
@@ -152,7 +152,9 @@ export const getBlock = createSelector(
 		return {
 			...block,
 			attributes: getBlockAttributes( state, clientId ),
-			innerBlocks: getBlocks( state, clientId ),
+			innerBlocks: areInnerBlocksControlled( state, clientId )
+				? []
```

Comment:
Should we use the `EMPTY_ARRAY` here?

Reply:
Sure! I'm not familiar with how much of a difference it would make in this scenario, but I see that it is used all over the place in these selectors :)